SPECS/samba.spec

```diff
@@ -3251,9 +3315,34 @@ rm -rf %{buildroot}
 
 %endif # with_clustering_support
 
-%{?_cov_results_package}
-
 %changelog
+* Mon Apr 14 2025 Lucas Ravagnier <lucas.ravagnier@vates.tech> - 4.10.16-25.1
+- Update from CentOS 7
+- Replace some value in specfiles with the same as before
+- Disable vfs_cephfs because it was only necessary for fedora
```

Comment:
No need to mention it, as it comes naturally from the conditions placed in the upstream spec file.